website/data/alerts.yaml

```diff
@@ -23,4 +23,8 @@ new-streamlet-api:
   content: |
     As of version 0.16.0, Heron offers a new **Streamlet API** that you can use to write topologies in a more declarative, functional manner, without needing to specify spout and bolt logic directly. The Streamlet API is **currently in beta** and available for [Java](../../developers/java/streamlet-api). The Streamlet API for Python will be available soon.
 
-    More information on the Streamlet API can be found [below](#the-heron-streamlet-api).
\ No newline at end of file
+    More information on the Streamlet API can be found [below](#the-heron-streamlet-api).
+eco-api-beta:
+  title: The Heron ECO API is in beta
+  content: The Heron ECO API can be used to build and test topologies locally.  The API still needs some testing and feedback from the community to understand how we  should continue to develop ECO.
```

Comment:
@joshfischer1108 - API can be used for topologies running as well in cluster - correct?

Reply:
@kramasamy Yes, just a typo error. I will fix